src/components/Select/SuggestionsList/SuggestionsList.tsx

```diff
@@ -15,7 +15,7 @@ export interface Props<S> {
     /** suggestionToString(suggestion) should return a string to be displayed in the UI. e.g.: suggestion => suggestion.name */
     suggestionToString: (suggestions: S) => string;
     /** to be shown when no suggestions are available */
-    noSuggestionsPlaceholder: string;
+    noSuggestionsPlaceholder?: string;
```

Comment:
Why is this? (See CONTRIBUTE.md 'Type definitions for Props and States' section)

Reply:
discussed in Slack